clicommand/artifact_search.go

```diff
@@ -64,6 +64,8 @@ Format specifiers:
 
   %S    SHA1 checksum of the artifact
 
+	%t    SHA256 checksum of the artifact
```

Comment:
spaces rather than tab to match the other lines

```suggestion
  %t    SHA256 checksum of the artifact
```